source/EventDetail.html.md.erb

```diff
@@ -7,10 +7,10 @@ tags: []
 Event Details and Resources
 ------
 
-Downloadable
-<%= link_to "What? Where? When? Guide", "resources/Frostburn2016-WhatWhereWhenGuide.pdf" %>
+Downloadable <%= data.event.year %> Docs
+<%= link_to "What? Where? When? Guide", "resources/Frostburn#{data.event.year}-WhatWhereWhenGuide.pdf" %>
```

Comment:
These are going to fail until they are created. It may be better to have them point to placeholders or ???